app/views/shared/_status_table.html.erb

```diff
@@ -1,6 +1,6 @@
 <tr>
   <td><%= link_to("#{order.user.first_name}'s Order #{order.id}", user_order_path(order.user, order)) %></td>
-  <td><%= number_to_currency order.total_cost/100 %></td>
+  <td><%= number_to_currency order.total_cost/100.00 %></td>
```

Comment:
it seems like we're dividing by 100 all over the place in the views.  Maybe we could move this functionality to the project.rb model